### ISSUE TITLE
Add more maintainers :-)

### ISSUE THREAD

**Issue by larsbergstrom** [barosl/homu#122]
@barosl, would you be open to adding some more reviewers to homu? There are many of us who'd love to continue supporting, extending, and adding features while you are busy with other things, and we'd prefer not to do that work in a separate fork!

Feel free to DM me on irc.mozilla.org to chat (larsberg) if you'd like to chat about it more :-)

**Comment by edunham** [comment]
Just a quick update, we discussed this at the last Servo meeting and we're going to temporarily continue development in the Servo fork in order to merge changes we need for our production Homu instance. We really appreciate your work on Homu, and we hope to reunite the forks once you get back.

**Comment by barosl** [comment]
Sorry for the HUGE delay. There were some big personal obstacles in my life recently, and because of that I've been distracted from the open source projects for months. Now that the problem is resolved, I think I can go on. If I were to add more maintainers @Manishearth will be one of them.

I actually think testing new features on the Servo fork might be a good idea to test new features rapidly. I'm very poor at making decisions, and maybe I take pull requests _too_ seriously. So I try to check if the pull requests are in perfect state again and again, before actually merging them. I think this attitude is not very useful, and even think it's quite wasteful, but I cannot resist such urges.

One of the consequences of this is that it can take so much time for me to accept some pull requests, especially large ones. That's why I think testing them directly on the Servo fork may be beneficial. However, it is also true that maintaining two repositories costs is a bit more work. I'm again poor at decisions!

**Comment by larsbergstrom** [comment]
@barosl Welcome back! Please don't hesitate to reach out to us (or me personally - larsberg @ mozilla.com) if there's anything we can do to help.

I think we're happy to maintain a fork and help with landing them back in your main upstream repository, if you'd prefer that workstyle.

Also, @Manishearth mentioned you have some private test scripts that you run on homu before landing a PR. We could also help with automating those tests via Travis, if that would help make it easier to land things without fear that stuff will break!

Again, glad to see you back and hope that all is better for you!

**Comment by Manishearth** [comment]
Yeah, I'm okay with keeping a fork both so that we can field test features (which get upstreamed later) and also so that we can make Servo-specific changes.

Welcome back!

**Comment by cgwalters** [comment]
FWIW I could spend (and just have spent) some time helping with review.  I'm investigating using Homu for a work project and I could justify a small amount of time on maintenance.  I'm not sure I agree with the Servo fork approach.  How about instead, create a git branch called `stable` or whatever, then we iterate a lot in master?  It's git, one can always go back.

**Comment by cgwalters** [comment]
Should we move our PRs to the Servo fork?

**Comment by Manishearth** [comment]
Fine by me.

**Comment by nightpool** [comment]
Hey, I'm part of an open-source project that uses homu.io as part of our build infrastructure. It seems like over the past few weeks the hosted version has stopped working. Is there a plan to bring it back going forward?

tagging @erickt who expressed interest in homu.io

**Comment by Manishearth** [comment]
The rust community team is considering hosting their own instance for use by everyone, yes 😄
